### DISCUSSION TITLE
Following, For You - Unread Indicator

### DISCUSSION THREAD

**Discussion by ghost** [community/community#13710] [Other Feature Feedback, Questions, & Ideas]
If you're going to have "Following" and "For You" as separate tabs, they need "unread" indicators.  Every time I visit, I feel compelled to flip between the two tabs to see if anything new is posted.  It's cumbersome, and distracting.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

maybe this update might help you with that situation, because you can filter it
